tests/unit/test_charm.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Jenkins-k8s charm unit tests."""
+
+# pylint:disable=protected-access
+
+from functools import partial
+from typing import Any, Callable, cast
+from unittest.mock import MagicMock
+
+import pytest
+import requests
+from ops.charm import PebbleReadyEvent
+from ops.model import ActiveStatus, BlockedStatus, Container, StatusBase
+from ops.testing import Harness
+
+import jenkins as jenkins_src
+from charm import JenkinsK8SOperatorCharm
+from jenkins import LAST_EXEC, UPDATE_VERSION, unlock_jenkins
+
+from .helpers import make_relative_to_path
+from .types_ import ContainerWithPath
+
+
+def test_unlock_jenkins(
```

Comment:
Should this be in the `test_jenkins.py` file? It seems to test a function from that module?

Reply:
fixed, thanks!